### PR TITLE
Added support for FIPS

### PR DESCRIPTION
**Pull request by dima618**
**Feature**

**What is this PR about? / Why do we need it?**
Added support for configuring the efs-csi-driver to use FIPS when mounting file systems.

**What testing is done?** 
Manual testing: https://quip-amazon.com/ODxXAOdSxTpb/efs-csi-driverefs-utils-FIPS-Testing-Design#temp:C:aMH7252de3f2d0446938cc776ddf

### CODE REVIEW

**Comment by dima618** [comment]
Ran `go mod vendor`

File: pkg/cloud/cloud.go [hunk 1]
```diff
@@ -133,9 +133,12 @@ func createCloud(awsRoleArn string) (Cloud, error) {
 		return nil, fmt.Errorf("could not get metadata: %v", err)
 	}
 
+	efs_client := createEfsClient(awsRoleArn, metadata, sess)
```
Comment (wongma7) [inline pkg/cloud/cloud.go:136]: prefer camelCase for names in golang

**Comment by wongma7** [comment]
/approve

thank you

**Comment by wongma7** [comment]
will let @Ashley-wenyizha do final review and  drop the '/lgtm'

**Pull request merged**